packages/workflow/src/Interfaces.ts

```diff
@@ -2031,7 +2031,7 @@ export interface IWebhookResponseData {
 }
 
 export type WebhookResponseData = 'allEntries' | 'firstEntryJson' | 'firstEntryBinary' | 'noData';
-export type WebhookResponseMode = 'onReceived' | 'lastNode' | 'responseNode' | 'formPage';
```

Comment:
Could this break any existing workflows out there that still have the response mode set to `formPage` ?

Reply:
it should not, this was assigned dynamically in form trigger if form node in connected nodes
```
	if (hasNextPage) {
			responseMode = 'formPage';
		}
```